Providers/nxOMSAutomationWorker/automationworker/3.x/worker/importHelper.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# ====================================
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+# ====================================
+PY_MAJOR_VERSION = 0
+PY_MINOR_VERSION = 2
+
+CONFIGPARSER_KEY = "configparser"
+QUEUE_KEY = "queue"
+import sys
+
+def install_aliases():
+    """
+    Imports the relevant modules based on the python runtime version.
+    After calling this method the caller can import the latest packages without having to consider about the 
+    backward compatibility.
+
+    Currently supported packages:
+    1. ConfigParser
+    2. Queue
+
+    """
+    
+    version_info = sys.version
```

Comment:
is this file needed ?

Reply:
To support Python2 runbooks in python3 environment